chsdi/static/doc/source/services/sdiservices.rst

```diff
@@ -754,21 +754,21 @@ The GetCapabilites document provides informations about the service, along with
 Parameters
 **********
 
-Only the RESTFul interface ist implemented. No KVP and SOAP. You *have* to provide a value for the `timestamp` parameter, the keywords `current` or 
+Only the RESTFul interface is implemented. No KVP and SOAP. You *have* to provide a value for the `timestamp` parameter, the keywords `current` or 
 `default` are not supported for now.
 
 A request is in the form:
 
 ::
 
-    <protocol>://<ServerName>/<ProtocoleVersion>/<LayerName>/<Stylename>/<Time>/<TileMatrixSet>/<TileSetId>/<TileRow>/<TileCol>.<FormatExtension>
+    <Scheme>://<ServerName>/<ProtocoleVersion>/<LayerName>/<Stylename>/<Time>/<TileMatrixSet>/<TileSetId>/<TileRow>/<TileCol>.<FormatExtension>
 
 with the following parameters:
 
 ===================    =============================   ==========================================================================
 Parameter              Example                         Explanation
 ===================    =============================   ==========================================================================
-Protocol               http ou https                   
+Scheme                 http or https                   The scheme type
```

Comment:
Wmts and https and clients : please double check with beat Tschanz

On Tue, Dec 8, 2015 at 2:48 PM, Gasser Loïc <notifications@github.com>
wrote:

>>  
>>  with the following parameters:
>>  
>>  ===================    =============================   ==========================================================================
>>  Parameter              Example                         Explanation
>>  ===================    =============================   ==========================================================================
>> -Protocol               http ou https                   
>> +Scheme                 http or https                   The scheme type
> Ok then it's all good, this comment was actually under WMTS instead of Terrain Service.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/geoadmin/mf-chsdi3/pull/1794/files#r46954076